### PR TITLE
Linting + prettying

### PR DESCRIPTION
**Pull request by jeffreyhshapiro**
**Description**
* Adds linting and prettying to the project
* Per [this thread](https://github.com/Anon-Inu/poc-network/issues/6#issuecomment-985476066) we're going to make all eslint issues warnings so commits aren't blocked by errors
* For anyone with `VSCode` there is a workplace config that allows you to run prettier on save

**Requirements**
* Husky requires > Git 2.9- you will need to update if you are running a lower version
* For VSCode users- install the `esbenp.prettier-vscode` extension

**To Test**
* Pull the branch and run `npm install`. Make a fake commit and hopefully you see the linter and prettier run.
* If you have `VSCode`, make some code changes and save. It should reformat

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (shamoons) [inline .vscode/settings.json:1]: Can we `.gitignore` this file?

File: package-lock.json [hunk 2]
```diff
@@ -0,0 +1,5105 @@
+{
+  "name": "poc-network",
```
Comment (shamoons) [inline package-lock.json:2]: Currently, I have it set up with `pnpm`instead of `npm`. Happy to switch through. Thoughts? cc @kfaria

**Review by shamoons — CHANGES_REQUESTED** [review]
A few small comments / questions

File: package-lock.json (hunk 2 above)
Comment (jeffreyhshapiro) [reply package-lock.json:2]: oooo i havent heard of `pnpm` which is which why didnt use it! i can switch to keep thinks consistent- the `pnpm-lock.yaml` file makes sense now though 😅

Comment (jeffreyhshapiro) [reply package-lock.json]: @shamoons installed with pnpm, package-lock no longer exists 🙏

File: .vscode/settings.json (hunk 1 above)
Comment (jeffreyhshapiro) [reply .vscode/settings.json:1]: do you want this completely excluded or just ignored from here on out?

Comment (shamoons) [reply .vscode/settings.json:1]: Gif ignore the whole directory since people will have different editor settings

Comment (jeffreyhshapiro) [reply .vscode/settings.json]: done

**Review by shamoons — APPROVED** [review]

**Comment by jeffreyhshapiro** [comment]
@kfaria any thoughts/feedback before this gets merged in?

**Comment by shamoons** [comment]
Merged - any issues that come up, we'll bug it

**Pull request merged**